servicex/resources/servicex_resource.py

```diff
@@ -68,6 +71,28 @@ def _validate_user():
             print(exc_value)
             return False, str(exc_value)
 
+    @staticmethod
+    def _verify_slack_request(req):
```

Comment:
This really needs documentation in the README, and some updates to `app.conf.template`
How do I set up slack correctly for this? Where do I get SLACK_SIGNING_SECRET?

Reply:
It's documented in ssl-hep/ServiceX#158, but I can update the template and move that over to this README where it's probably a better fit.